commonItems/UnitTests/CommonFunctionsTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace commonItems.UnitTests {
+    public class CommonFunctionsTests {
+        [Fact]
+        public void LastDigitOneGivesSt() {
+            Assert.Equal("st", CommonFunctions.CardinalToOrdinal(1));
```

Comment:
> Static equals overloads are used for comparing instances of two types for reference equality. **This method should not be used for comparison of two instances for equality**. This object will always throw with Assert.Fail. Please use Assert.AreEqual and associated overloads in your unit tests.  

(emphasis mine)

Equal -> AreEqual

Same for all tests. Might have to go back and redo existing tests.

Reply:
Isn't this from MSUnit documentation? I use the XUnit framework.

Reply:
Oh, never mind then.